README.md

```diff
@@ -48,6 +49,7 @@
 * https://pub.dev/packages/flutter_redux
 * https://pub.dev/packages/flutter_mobx
 * https://pub.dev/packages/flutter_bloc
+* https://flutter.dev/docs/development/data-and-backend/state-mgmt/options // You Have Multiple choice in FLUTTER 
```

Comment:
I don't think this is a good source, neither it is applicable in this context.

Reply:
That's an interesting link. I like that there are many options described, however, for junior devs it might be too much information and it might not be clear what option to go with.